packages/pyright-internal/src/languageServerBase.ts

```diff
@@ -1152,6 +1155,31 @@ export abstract class LanguageServerBase implements LanguageServerInterface, Dis
         return result;
     };
 
+    protected onSaveTextDocument = async (params: WillSaveTextDocumentParams) => {
+        const fileUri = Uri.file(params.textDocument.uri, this.serviceProvider);
+        const workspace = await this.getWorkspaceForFile(fileUri);
+        const rootUri = workspace.rootUri!;
+        const previouslyBaselinedErrors = getBaselinedErrorsForFile(this.fs, rootUri, fileUri);
+        const diagnosticsForFile = this.documentsWithDiagnostics[params.textDocument.uri];
+        let filesWithDiagnostics: FileDiagnostics[];
+        if (diagnosticsForFile) {
```

Comment:
this incorrectly removes baselined errors when files are saved from a vscode find/replace

Reply:
https://github.com/DetachHead/basedpyright/pull/608/commits/942bff39dd011c97f5b2f840d68c78bbb715856d